doc/design/requirements.md

```diff
@@ -0,0 +1,69 @@
+# Requirements and goals
+
+Includes the overlapping projects of TiUp, tiup-cluster, and tiup-playground.
+
+We should distinguish between goals and requirements for the long term vs an initial release.
+
+## Vision and high-level goals
+
+* Manage and deploy TiDB and TiKV clusters
+* Open source project, with a community of users and contributors
+* Help TiDB and TiKV developers run and test their code in a production-like environment
+* Make TiDB an easy to use database
+* Be the standard way for TiDB users to manage their clusters
+* Make it easy to try out TiDB, both on a single machine and a cluster
+* Extensible
+
+## Requirements
+
+### Manage components
+
+* Install and update from a remote or local server, or from user-provided binaries
+* Uninstall/remove
+* Provide information about installed components and how to use them to the user
+
+### Self-update
+
+* Update TiUp from remote server
+* Uninstall
+
+### Deployment and managing a deployed cluster
+
+TODO
```

Comment:
There some documents but haven't been translated to English, I think we can link to the documents after we finished translating.